src/endpoint/endpoint.js

```diff
@@ -41,6 +41,8 @@ const background_scheduler = require('../util/background_scheduler').get_instanc
 const endpoint_stats_collector = require('../sdk/endpoint_stats_collector');
 const { NamespaceMonitor } = require('../server/bg_services/namespace_monitor');
 const { SemaphoreMonitor } = require('../server/bg_services/semaphore_monitor');
+const { CertWorker } = require('./cert_worker');
```

Comment:
let's keep in the same place with all the bg_workers... or at least under endpoint/bg_services/
but maybe we will use it in core as well in the future